node/src/chain/fsm.rs

```diff
@@ -316,10 +319,14 @@ impl<N: Network, DB: database::DB, VM: vm::VMExecution> SimpleFSM<N, DB, VM> {
             );
```

Comment:
the above `event` message is wrong. The actual event here is that we found the Candidate corresponding to Quorum.

Reply:
We have the "src" field to distinct between those cases

Reply:
My point is that we didn't receive a block... we just found one in the DB that corresponds to the quorum message.